examples/minimal/.gitignore

```diff
@@ -0,0 +1,2 @@
+index.js
```

Comment:
This seems weird to me. We should just build to a `dist` folder so we can ignore that.

Reply:
I think it would get a bit messy because of the separate dependencies

Reply:
I agree with @sachinraja, built JS files shouldn't be alongside source code, better to have them in some sort of folder like `dist` or `build`.

Reply:
I was wrong, it wasn't a mess 😅